### ISSUE TITLE
Works fine on MacOS Sierra

### ISSUE THREAD

**Issue by dragonfax** [sg3des/2048#1]
You just need to build it with 
`go build -ldflags=-s`

Otherwise you get `killed: 9`.

**Comment by dragonfax** [comment]
![2048](https://user-images.githubusercontent.com/933806/27499159-63f28b84-5817-11e7-8570-63d6c914e36d.png)

**Comment by dragonfax** [comment]
```
ZiggyPiggy:2048 jstillwell$ go build
ZiggyPiggy:2048 jstillwell$ ./2048
Killed: 9
```

**Comment by sg3des** [comment]
thx

**Comment by peterhellberg** [comment]
There is no need for `-ldflags=-s` atleast not on OS X El Capitan and High Sierra (Using Go 1.10)

**Comment by sg3des** [comment]
`-ldflags=-s` needed for reduced the size of a binary file, it is not necessary.

**Comment by sg3des** [comment]
same issue about `-ldflags=-s` - https://github.com/golang/go/issues/11887, if i`m right fixed in 1.8.1
